package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "surveyor",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
@rafayet-monon We can move `dependencies ` block to address before `devDependencies` for making it easier to see 😁